app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -255,7 +255,7 @@
     private Optional<Integer> pokemonCP = Optional.absent();
     private Optional<Integer> pokemonHP = Optional.absent();
     private double estimatedPokemonLevel = 1.0;
-    private @NonNull Optional<String> ssFile = Optional.absent();
+    private @NonNull Optional<String> screenShotPath = Optional.absent();
```

Comment:
Path is ambiguous -- whether it's dir or file. That's why I had used File. But not a big deal.